tests/bud.bats

```diff
@@ -1765,7 +1765,7 @@ _EOF
 
 @test "bud with preprocessor error" {
   target=alpine-image
-  run_buildah 0 bud -q $WITH_POLICY_JSON -t ${target} -f Error.in $BUDFILES/preprocess
+  run_buildah 0 bud $WITH_POLICY_JSON -t ${target} -f Error.in $BUDFILES/preprocess
```

Comment:
This is going to conflict with #4057

Reply:
Ok let's merge yours and then I will rebase